relayer-cli/src/commands/start.rs

```diff
@@ -59,9 +59,9 @@ impl Runnable for StartCmd {
 }
 
 /// Register the SIGHUP and SIGUSR1 signals, and notify the supervisor.
-/// - SIGHUP: Trigger a reload of the configuration.
+/// - [DEPRECATED] SIGHUP: Trigger a reload of the configuration.
 /// - SIGUSR1: Ask the supervisor to dump its state and print it to the console.
-fn register_signals(reload: ConfigReload, tx_cmd: Sender<SupervisorCmd>) -> Result<(), io::Error> {
+fn register_signals(_reload: ConfigReload, tx_cmd: Sender<SupervisorCmd>) -> Result<(), io::Error> {
```

Comment:
Should we also mark the `ConfigReload` struct or its `reload` method as `#[deprecated]` for extra caution around that module?

ref: https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute

Reply:
I don't think it's needed since we don't expect anyone to rely on that programatically.